rise/static/configurator.md

```diff
@@ -0,0 +1 @@
+# This is a placeholder for a markdown file that is displayed by nbextensions_configurator
```

Comment:
Do you want to use this placeholder for a sort of README?

Reply:
Yes, something like it; when you manage an extension in nbextensions, it shows the parameters, and below that a documentation segment
See for example autopep8 that is among the first ones to get a sense

I haven't spent any tome on that, it was just to materialize this possible place where we can publish information; it could also just be that some other  already available README could do the trick just as well; what do you think ?

Reply:
Looking at this again, in particular wrt the doc2 branch that comes just next, I think we could just use the toplevel's README.md for that purpose.  
This actually leads us back to this packaging thing; I have not yet looked into that, mostly because I feel competent only for pip; but I will give this a shot shortly, I'll just let you see the conda side of things if that's all right with you.

Reply:
After looking at setup.py, it seems to me that the pip packaging should come out OK.

Specifically there are now 6 files that should be installed in the `nbextensions/rise` area - wherever that is : 
* configurator.md
* `main.css`
* `main.js`
* `reveal.js`
* `rise.png`
* `rise.yaml`

I propose that we leave `configurator.md` as it is for now, and I will iron this out in the second PR - #347 - it will be easier there.

Reply:
oops, correction, the PR where this will be easier to deal with is #355 because I have introduced a meta target `npm run build` that would be the right place to add a copy of the toplevel README into the static area.

regardless, I suggest we keep this issue aside for now.